django/mgmt/static/js/collections.js

```diff
@@ -17,7 +17,7 @@ function coord_handler(response) {
 
 function experiment_handler(response) {
     var resources = get_resource_names();
-    var detail_base_url = window.location.href + "/";
+    var detail_base_url = API_ROOT + "mgmt/resources/" + resources[0] + "/";
```

Comment:
will these URLs ever change? maybe makes more sense to just do a `window.location.href` with a right-side slash trim, and then add the slash:

```js
// remove trailing slashes, and guarantee there's only one at the end
window.location.href.replace(new RegExp("\/+$"),"") + "/"
```

that way it's portable to any route(?)
idk if that's a risk here

Reply:
Good point. By other route do you mean other APIs like api.bossdb.io vs api.theboss.io? The reason I came up with this solution is because the `delete_base_url` and `neuroglancer_base_url` use the same URL formatting. I figured it would make sense to keep them all consistent.